modules/arm_plugin/tests/functional/shared_tests_instances/set_device_name.cpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <string>
+#include <stdexcept>
+
+#include "set_device_name.hpp"
+
+namespace ov {
+namespace test {
+void set_device_suffix(const std::string& suffix) {
+    if (!suffix.empty()) {
+        throw std::runtime_error("The suffix can't be used for ARM CPU device!");
```

Comment:
The same as for `nvidia_plugin`, please disable test if it will fail after merge